src/components/markdown.tsx

```diff
@@ -572,6 +575,8 @@ function Image(props: React.ComponentPropsWithoutRef<"img">) {
 }
 
 function Code({ className, inline, children }: CodeProps) {
+  const { copy } = useClipboard()
```

Comment:
We shouldn't need this `useClipboard()` hook. We're already using a `copy-to-clipboard` package in other parts of the code base:

https://github.com/lumen-notes/lumen/blob/main/src/components/note-card.tsx#L3